tests/conftest.py

```diff
@@ -78,7 +79,8 @@ def get_autoscaling():
             {'InstanceId': 'ins-2', 'LifecycleState': 'InService'},
             {'InstanceId': 'ins-3', 'LifecycleState': 'InService'},
             {'InstanceId': 'ins-4', 'LifecycleState': 'unknown'},
-        ]
+        ],
+        'CreatedTime': datetime(2018, 6, 6, 9, 59, 38, 127000, tzinfo=tzutc())
```

Comment:
This is how it is returned from https://github.com/zalando-zmon/zmon-aws-agent/blob/b2877e63a02b92e3e64d76cef6dd22b2839c1ed5/zmon_aws_agent/aws.py#L491-L496